src/vivarium/framework/population/population_view.py

```diff
@@ -41,7 +41,6 @@ def __init__(
         manager: PopulationManager,
         component: Component | None,
         view_id: int,
-        query: str = "",
```

Comment:
The only way to set a pop view default query is to call `set_default_query()` now